<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-576CM0N" ignored="3" name="TS_WEB_004 :: To verify 2 GB/Day Packs pricing of plans, validity and benefits Negative flow." tests="1" failures="1" timestamp="2020-09-24T10:38:47 IST" time="25.151" errors="0">
  <testcase name="jio_verify2GBPlansNegativeFlow" time="7.821" classname="com.TBlocks_POC.TblocksIndex">
    <failure type="java.lang.AssertionError" message="Test Verification failed, please check test logs.">
      <![CDATA[java.lang.AssertionError: Test Verification failed, please check test logs.
at org.testng.Assert.fail(Assert.java:99)
at com.TBlocks_POC.TblocksIndex.jio_verify2GBPlansNegativeFlow(TblocksIndex.java:297)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- jio_verify2GBPlansNegativeFlow -->
  <testcase name="jio_verify2GBPlansPositiveFlow" time="0.0" classname="com.TBlocks_POC.TblocksIndex">
    <ignored/>
  </testcase> <!-- jio_verify2GBPlansPositiveFlow -->
  <testcase name="zain_verifyLogin_Dashboard_Price" time="0.0" classname="com.TBlocks_POC.TblocksIndex">
    <ignored/>
  </testcase> <!-- zain_verifyLogin_Dashboard_Price -->
  <testcase name="test2" time="0.0" classname="com.utils.ZainProvider">
    <ignored/>
  </testcase> <!-- test2 -->
</testsuite> <!-- TS_WEB_004 :: To verify 2 GB/Day Packs pricing of plans, validity and benefits Negative flow. -->
